Tools/Scripts/webkitpy/common/checkout/checkout_mock.py

```diff
@@ -104,6 +104,11 @@ def __init__(self):
         self._filesystem = MockFileSystem()
 
     def commit_info_for_revision(self, svn_revision):
+        if isinstance(svn_revision, str) and svn_revision.startswith('r'):
+            svn_revision = svn_revision[1:]
+        if isinstance(svn_revision, str) and not svn_revision.isdigit():
+            return None
```

Comment:
why none here?

Reply:
or more to the point will this fail silently or with a weird error message

Reply:
This is a mock class. There is already a comment in this function that says this:
```
# any "unrecognized" svn_revision will return None.
```
just wasn't in the diff.